3Dmol/glviewer.js

```diff
@@ -1202,6 +1206,32 @@ $3Dmol.GLViewer = (function() {
             ret.normalize();
             return ret;
         };
+        var spinInterval;
+        this.spin = function(spin,axis,interval){
```

Comment:
I was also thinking that the axis could define speed as well [2,0,0] would spin twice the default on the x axis as [1,0,0]. With just supplying the interval you can only specify one speed for all of the axis and with a high interval there is visible skipping. This way it is continuous and you can specify different speeds for all 3 axis.